### DISCUSSION TITLE
Test Hesabı, Test Kartları, Dokumantasyonlar

### DISCUSSION THREAD

**Discussion by mhmtpacal** [mewebstudio/pos#246] [General]
**Bu kütüphanenin desteklediği bütün bankaları deneyerek sistemimize ekliyoruz fakat test kart ve hesapları gerekiyor birkaçını bulabildik fakat elinde olanlar yorum bırakırsa konuyu güncellerim, birçok kişinin işine yarayacağını düşünüyorum;**

- Akbank Yeni API:
    - destek email: akbanksanalposdestek@akbank.com
    - Dokumantasyon: 
         - https://sanalpos.akbank.com/#entry 
- Param POS
    - destek email: entegrasyon@param.com.tr
    - Dokumantasyon
         - https://dev.param.com.tr/
- TEB :
- Payten (asseco) :
    - Dokumantasyon: 
        - v1.2 (31.01.2024)
           [Nestpay_Entegrasyon_Dokumani_3D ver3.pdf]. 
            (https://github.com/mewebstudio/pos/files/14418533/Nestpay_Entegrasyon_Dokumani_3D.ver3.pdf)

- İşbank : 
- Şekerbank :
- Halkbank : 
- Finansbank (PayFor) : 
   - Dokumantasyon: https://vpos.qnbfinansbank.com/Help/apiIntegrationDocument
   - destek email: sanalpos@qnbfinansbank.com
- Ziraat Katılım (PayFor) : 
   - destek email: vpos_operasyon@paycore.com
- Ziraat : Payten (AssecoPos)
[Nestpay_Entegrasyon_Dokumani_PayHosting.pdf](https://github.com/mewebstudio/pos/files/14705209/Nestpay_Entegrasyon_Dokumani_PayHosting.pdf)
[Nestpay_Entegrasyon_Dokumani_3D__Pay.pdf](https://github.com/mewebstudio/pos/files/14705211/Nestpay_Entegrasyon_Dokumani_3D__Pay.pdf)
[Nestpay_Entegrasyon_Dokumani_3D_PayHosting.pdf](https://github.com/mewebstudio/pos/files/14705215/Nestpay_Entegrasyon_Dokumani_3D_PayHosting.pdf)
[Nestpay_Entegrasyon_Dokumani_3D.pdf](https://github.com/mewebstudio/pos/files/14705218/Nestpay_Entegrasyon_Dokumani_3D.pdf)
[Nestpay_ZiraatBankasi_Test_Bilgileri-.pdf](https://github.com/mewebstudio/pos/files/14705222/Nestpay_ZiraatBankasi_Test_Bilgileri-.pdf)

- Vakıfbank VPOS 7/24 MPI (PayFlex V4 MPI):
    - destek email: sanalpos@vakifbank.com.tr
    - v1.0 (16.07.2019)
        - [Dokumanlar ve ornek kodlar](https://www.vakifbankpos.com.tr/assets/files/VPOS%20724%20_GET724_ENTEGRASYON_DOKUMANI.zip)

- Vakıf bank GÜVENLİ ORTAK ÖDEME (PayFlexCPV4Pos): 
    - Dokumantasyon: 
        - Güncel örnek kodlar ve dokümanlar: https://www.vakifbankpos.com.tr/SanalPosKilavuzlari 
        - v2.0 (05.06.2023) [GÜVENLİ ORTAK ÖDEME (3DPAY) ENTEGRASYON DOKUMANI ve Örnek Kodlar](https://www.vakifbankpos.com.tr/assets/files/GUVENL%C4%B0%20ORTAK%20%C3%96DEME%20ENTEGRASYON%20DOKUMANLARI.zip)

- İşbank : 
- YapıKredi (PosNet):
     - Dokumantasyon 
          - v2.0.1.5 (10.01.2023) [POSNET-3D-Secure-Entegrasyonu-2.0.1.5.pdf](https://github.com/mewebstudio/pos/files/15379993/POSNET-3D-Secure-Entegrasyonu-2.0.1.5.pdf)

- Albaraka Türk : 
- Kuveyttürk : 
    - destek email: sanalposdestek@kuveytturk.com.tr
    - TDV2.0.0
        - [3DSecure Model.pdf](https://github.com/mewebstudio/pos/files/15128974/3DSecure.Model.pdf)
        - [Iptal_Iade_KismiIade.pdf](https://github.com/mewebstudio/pos/files/15129055/Iptal_Iade_KismiIade.pdf)
        - [Non3D.Model.pdf](https://github.com/mewebstudio/pos/files/15129067/Non3D.Model.pdf)

- Deniz bank : 

- Vakıf Katılım Bankası
    -  Dokumantasyon
        - [v2.8 Sanal Pos Entegrasyon Dokümanı](https://github.com/mewebstudio/pos/files/14228216/GuvenliOdemeSayfasiEntegrasyonDokumani.pdf)
        - [v.2.7 Sanal Pos API Entegrasyon Dökümanı](https://github.com/mewebstudio/pos/files/14228217/SanalPOSApiEntegrasyonDokumani.pdf)
        - [v2.1 Sanal Pos Uygulaması Kullanım Kılavuzu](https://github.com/mewebstudio/pos/files/14228218/SanalPOSKullanimKilavuzu.pdf)
- AkÖde/Tosla
    -  Dokumantasyon
        - https://tosla.com/isim-icin/gelistirici-merkezi
    - Iletisim: sanalposdestek@akode.com

- Izyico
    -  Dokumantasyon
        - https://docs.iyzico.com/
    - Test Kartlar https://docs.iyzico.com/ek-bilgiler/test-kartlari

PayTR
     -  Dokumantasyon
        - https://dev.paytr.com/
    - Test Kartlar https://dev.paytr.com/direkt-api/test-kart-bilgileri

**Comment by nuryagdym** [comment]
Merhabalar,
Dusunceni guzel, bankalarin bu bilgileri iceren web sayfalarin linkleri varsa ekleyebiliriz.
Ancak acik bir sekilde paylasilmayanlari eklememiz dogru olmaz, bankalar paylasildigini fark edince degistirmek isteyebilirler.

**Comment by mhmtpacal** [comment]
> Merhabalar, Dusunceni guzel, bankalarin bu bilgileri iceren web sayfalarin linkleri varsa ekleyebiliriz. Ancak acik bir sekilde paylasilmayanlari eklememiz dogru olmaz, bankalar paylasildigini fark edince degistirmek isteyebilirler.

Test kart ve hesaplarının çok bir önemi olduğunu düşünmüyorum ama tabi varsa nedeni, kapatabilirim konuyu

**Comment by nuryagdym** [comment]
Konu acik kalabilir, musait oldugumda elimdeki bilgileri eklerim.
Web sitelerinde veya dokumantasyonlarinda paylasilmissa, burda paylasabiliriz.
Ama dedigim gibi, banka acik bir sekilde paylasmiyorsa bir sebebi vardir ve burda paylasilmasi sorun cikarabilir.

**Comment by crazyshark2** [comment]
İşbankası güncel test bilgileri
[İşbank_Sanal_POS_Test_Bilgileri1.pdf](https://github.com/user-attachments/files/18541734/Isbank_Sanal_POS_Test_Bilgileri1.pdf)

**Comment by esakarya** [comment]
Merhaba param gibi tr de fazlasıyla yaygın olan iyzico yu eklemeyi düsünürmüsünüz?

https://docs.iyzico.com/

**Reply by nuryagdym** [reply]
Olabilir, iyzico kendi basina bir sistem oldugu icin gerek degildir diye dusunuyordum.

Issue acip bir oylama isterim, duruma gore yapabilirim

**Comment by esakarya** [comment]
Türkiyede sanal pos, param iyzico paytr 3 ü üzerinde dönüyor. İş bankasının moka var birde ama bunlar kadar populer degil. Paytr yide tavsiye etmem, hemen sonuc dönmuyor, kuyruga aliyor. İşlemin sonlandirdiktan sonra sonuç dönüyor. Toslada piyasaya girmeye calistigi icin yeni musterilere iyi oran veriyor.  Digerini kullanan sayisi az hepsipay vs.
